src/test/java/com/wikia/webdriver/testcases/auth/ForcedLoginTests.java

```diff
@@ -46,11 +46,18 @@ public void anonCanLogInViaUserLoginPage() {
     assertTrue(base.isStringInURL(URLsContent.SPECIAL_UPLOAD));
   }
 
+  @Test(invocationCount = 5)
   public void anonCanLogInOnSpecialWatchListPage() {
     WikiBasePageObject base = new WikiBasePageObject();
     base.openSpecialWatchListPage(wikiURL);
     base.clickLoginOnSpecialPage();
     new AttachedSignInPage().login(user);
+    //TODO: To delete
+    try {
+      Thread.sleep(2000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)